test/e2e/utils/string_utils.py

```diff
@@ -3,7 +3,7 @@
 
 
 def random_string(size=1000) -> str:
-    return (
+    return random.choice(string.ascii_letters) + (
```

Comment:
It's better to call `.strip()` on the resulting string as `pinnwand` strips both leading and trailing whitespace.

Reply:
done